grapher/core/GrapherConstants.ts

```diff
@@ -42,6 +42,11 @@ export enum FacetStrategy {
     column = "column", // One chart for each Y column
 }
 
+export enum FacetScaleMode {
+    relative = "relative", // all facets have their own y range
+    absolute = "absolute", // all facets share the same y range
+}
```

Comment:
Hm, the relative/absolute terms don't make the right neurons fire in my head. 

Are there some other terms that could describe the same thing? Just throwing some terms: `individual`, `none`, `independent` maybe? And `uniform`, `global`, `identical`, `shared`... not exactly sure. 

Could also go for something longer and more descriptive. The comments describe it really well.

Reply:
Agreed. I like `independent` and `shared`, I'll switch to those. Note that `uniform` comes from the visible toggle label "Uniform y-axis". I'll leave the label text the same.